packages/blade/src/components/Table/Table.web.tsx

```diff
@@ -504,15 +595,19 @@ const _Table = <Item,>({
           )}
           {toolbar}
           <StyledReactTable
-            role="table"
-            layout={{ fixedHeader: shouldHeaderBeSticky, horizontalScroll: true }}
+            role={isVirtualized ? 'grid' : 'table'}
+            layout={{ fixedHeader: true, horizontalScroll: true }}
             data={data}
             // @ts-expect-error ignore this, theme clashes with styled-component's theme. We're using useTheme from blade to get actual theme
             theme={tableTheme}
             select={selectionType !== 'none' ? rowSelectConfig : null}
             sort={sortFunctions ? sort : null}
             $styledProps={{
-              height,
+              height: isVirtualized ? height || `${VirtualizedTableDimensions.height}px` : height,
```

Comment:
```suggestion
              height: height ?? `${VirtualizedTableDimensions.height}px`
```

is it same thing? because previously it was always passing `height` no?

Reply:
this will break since. if height is not present it will pass undefinedpx in normal table